reinforcement_learning/Lesson-4-Policy-Based-Learning-Method-draft.ipynb

```diff
@@ -0,0 +1,403 @@
+{
```

Comment:
<p>Rephrase to be supportive: Instead of "should be trivial..." something like "Let's see how well you've been following along, is the action space here <strong>discrete</strong> or <strong>continuous</strong>?"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>